### ISSUE TITLE
`TypedDict` attribute suggestions don't recognise f-string quote type

### ISSUE THREAD

**Issue by DanielNoord** [microsoft/pylance-release#1919]
## Environment data

-   VS Code version: 1.61.0
-   Extension version (available under the Extensions sidebar): 43.247.711
-   OS and version: macOS 11.6
-   Python version (& distribution if applicable, e.g. Anaconda): 3.9.7
-   Type of virtual environment used (N/A | venv | virtualenv | conda | ...): pyenv
-   Value of the `python.languageServer` setting: Default, `Pylance` I guess?

## Actual & Expected behaviour

Currently when writing an f-string started by `"` and trying to access an attribute of a `TypedDict` the extension automatically suggests the attributes found on that `TypedDict`. However, when choosing one of them, they are inserted automatically with `"`. It would be better if the extension picks up the f-string is demarcated by `"` and adds the attribute with `'`.

As this is actually breaking the code you are writing I filed this as a bug report, but you could also consider this a feature request. Sorry if this was incorrect!

## Steps to reproduce:

Code to reproduce:
```python
from typing import TypedDict


class TestDict(TypedDict):
    attribute: str

MY_DICT = TestDict(attribute="test")

print(f"Let me print this {MY_DICT[]}")
```
![ezgif-1-2694d7b4f584](https://user-images.githubusercontent.com/13665637/136528859-c08698e4-4684-434f-bdfc-19d7ca5459f2.gif)

**Comment by michaeloliverx** [comment]
I was about to create a new issue for this exact thing. It would be a good quality of life improvement if this could be fixed :)

**Comment by bschnurr** [comment]
This issue has been fixed in version 2022.5.1, which we've just released. You can find the changelog here: [CHANGELOG.md](https://github.com/microsoft/pylance-release/blob/main/CHANGELOG.md#202251-11-May-2022)
